rules/windows/file/file_event/file_event_win_create_evtx_non_common_locations.yml

```diff
@@ -1,12 +1,15 @@
 title: EVTX Created In Uncommon Location
 id: 65236ec7-ace0-4f0c-82fd-737b04fd4dcb
 status: experimental
-description: Detects the creation of new files with the ".evtx" extension in non-common locations. Which could indicate tampering with default evtx locations in order to evade security controls
+description: |
+    Detects the creation of new files with the ".evtx" extension in non-common or non-standard location.
+    This could indicate tampering with default evtx locations in order to evade security controls or simply exfiltration of event log to search for sensitive information within.
+    Note that backup software and legitimate administrator might perform similar actions during troubleshooting.
```

Comment:
```suggestion
    Detects the creation of new files with the ".evtx" extension in non-common or non-standard location.
    This could indicate tampering with default EVTX locations in order to evade security controls or simply exfiltration of event log to search for sensitive information within.
    Note that backup software and legitimate administrator might perform similar actions during troubleshooting.
```